proposals/0091-SDLScreen-SDLWindow-Projection.md

```diff
@@ -1,110 +1,209 @@
 # SDLCarWindow Video Projection Developer Interface
 
 * Proposal: [SDL-0091](0091-SDLScreen-SDLWindow-Projection.md)
-* Author: [David Switzer](https://github.com/davidswi)
-* Status: **Returned for Revisions**
+* Author: [Michael Pitts](mailto:mpitts@xevo.com)
```

Comment:
Can you please update this to your github account?

Reply:
Done